pages/index.tsx

```diff
@@ -33,7 +33,7 @@ const Home: NextPage = () => {
 }
 
 export default Home
-export async function getStaticProps({ locale }) {
+export async function getStaticProps({ locale }: { locale:string }) {
```

Comment:
Oq acha de mantermos interface aqui?

```suggestion
type StaticPropsTypes = { locale: string }

export async function getStaticProps({ locale }: StaticPropsTypes) {
```